mtags-interfaces/src/main/java/scala/meta/pc/PresentationCompiler.java

```diff
@@ -179,6 +179,10 @@ public CompletableFuture<List<SyntheticDecoration>> syntheticDecorations(Synthet
 		return CompletableFuture.completedFuture(Collections.emptyList());
 	}
 
+	public CompletableFuture<List<PcSymbolInformation>> info(String symbol) {
```

Comment:
When can it return multiple? Is there a known case?

Reply:
overloaded methods

Reply:
Though usually they have a different symbol

method()
method(+1)
etc.

so this should not repeat.

Reply:
Yes, but I want it to return all. When we search for implementations of methods, we find implementations of owner class/trait and only then implementations of method, needs to be this way since we index only class hierarchy. E.g. if `class A extends B` and we're looking for implementations of `B.method(+1)`, the searched method will be `A.method` but we don't know the disambiguator so I get them all to find the one I need. In `Compilers.scala` there is `info` and `infoAll`, the first one does exactly what you'd expect and returns only info about that exact symbol.

If this is too odd, we can add `overloadedSymbols` to `PcSymbolInformation` to learn all the overloaded and get info about each one by one.

Reply:
@tgodzik, would you prefer I change this?

Reply:
Could we get a full method symbol ? This should be possible using Mtags class. :thinking: